src/lib/utils/thorchain/index.ts

```diff
@@ -1,6 +1,5 @@
-import type { AccountId, ChainId } from '@shapeshiftoss/caip'
+import type { AccountId, AssetId, ChainId } from '@shapeshiftoss/caip'
```

Comment:
re: these guys, since it's the LSP doing the previous flavor, we should probably enforce things out at lint/tsconfig level if that's OCD-triggering!

Reply:
we can look for a rule if there is a more explicit one, however, mine follows this formatting when doing the eslint fix both from package.json and vscode lint which leads me to believe it is some vim lsp business.